tests/reframe/eessi-checks/prgenv/mpi.py

```diff
@@ -0,0 +1,20 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+# Try to use an import to define all site-specific things
+# import system_properties
+
+@rfm.simple_test
+class MpiHelloWorld(rfm.RegressionTest):
+    def __init__(self):
+        # We don't define these here to keep tests generic
+        # Sensible systems & programming environments should be defined in your site configuration file
+        self.valid_systems = ['*']
+        self.valid_prog_environs = ['*']
```

Comment:
shouldn't we need to set this to load openmpi and gcc; it will not have anything by default

Reply:
All of that is provided by loading the module. A programming environment is only needed if you want to run tests that e.g. compile and run code. Then, you can define a programming environment to load a fixed collection of modules, and test that collection. (think about hello-world tests where you test e.g. GCC and intel compilers, some basic flags, etc)